src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const RENT_PRICE = 40;
+  const DAY_7 = 7;
+  const DAY_3 = 3;
+  const DISCOUNT_AFTER_DAY_7 = 50;
+  const DISCOUNT_AFTER_DAY_3 = 20;
+  let rent = RENT_PRICE * days;
 
+  if (days >= DAY_7) {
+    rent -= DISCOUNT_AFTER_DAY_7;
+  } else if (days >= DAY_3) {
+    rent -= DISCOUNT_AFTER_DAY_3;
+  }
```

Comment:
You have used `let` for the variable `rent`. However, you can simplify your code by using `const` instead and returning the result directly from the `if` conditions. This will also make your code more readable and maintainable.